Readme.md

```diff
@@ -34,9 +34,12 @@ test:
 ### Create additional database(s)
     rake parallel:create
 
-### Copy development schema (repeat after migrations)
```

Comment:
also needs to be ` (repeat after migrations)`

Reply:
ah yes, will update ty